src/components/input-panel/compiled-spec-header/index.tsx

```diff
@@ -73,6 +127,34 @@ class CompiledSpecDisplayHeader extends React.PureComponent<Props> {
             </>
           )}
 
+          {this.props.sidePaneItem === SIDEPANE.Config && (
+            <>
+              <ArrowUpCircle
+                data-tip
+                data-for="mergeConfig"
+                onClick={e => {
+                  e.stopPropagation();
+                  this.handleMergeConfig(e);
+                }}
+              />
+
+              <ArrowDownCircle
```

Comment:
Can we make these actual `<button>` tags so that screenreaders can better find them?